src/rules/converters/strict-boolean-expressions.ts

```diff
@@ -0,0 +1,37 @@
+import { RuleConverter } from "../converter";
+
+export const ForbiddenOtherNonBooleanTypes =
+    "String, number, enum and mixed union types are forbidden.";
```

Comment:
It would be a little confusing to get this general error if you didn't already have some of those options enabled. It should really be specific to the input rule arguments.

For example, if you allowed number, enum, and mixed union types, you should probably get:

```
String type comparisons are now forbidden.
```

Reply:
The problem is that neither of those types are allowed, no matter if you set them on the tslint rule. The eslint doesn't allow it, only the `null` and `undefined` types are.

Reply:
Ok dokes, you've convinced me. 😄